packages/core/src/controllers/SwapController.ts

```diff
@@ -202,7 +202,7 @@ export const SwapController = {
       invalidSourceTokenAmount,
       availableToSwap:
         caipAddress && !invalidToToken && !invalidSourceToken && !invalidSourceTokenAmount,
-      isAuthConnector: type === 'AUTH'
+      isAuthConnector: type === 'ID_AUTH'
```

Comment:
was 'AUTH' stil a valid type?

Reply:
`AUTH` and `ID_AUTH` shares the same type as `ConnectorType`

authConnectorId = `ID_AUTH`
authConnectorType = `AUTH`

This is confusing, so we should definitely improve this